conf/CodeStyle/IntelliJ/import

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+_CALLING_DIR="$(pwd)"
+
+jar cf AlluxioCodeStyle.jar codestyles/AlluxioStyleCopy2.xml options/code.style.schemes.xml
```

Comment:
make these absolute paths, use the `-` flag to ensure its read properly across various shell implementations